### DISCUSSION TITLE
Show size of bundles in percentage

### DISCUSSION THREAD

**Discussion by sayedsaeedhosseini** [webpack-contrib/webpack-bundle-analyzer#575] [Ideas]
#### Issue description
Show size of modules in percentage in compared with total bundles

**Comment by Hosein-mh** [comment]
same issue

**Comment by AntonioRedondo** [comment]
This would be very useful!

Also, in addition to the fantastic default chart, showing the same information in the form of a hierarchical table would be a great addition.

Something like the file list at the top of the squares' chart in [WinDirStat](https://en.wikipedia.org/wiki/WinDirStat):

![image](https://user-images.githubusercontent.com/5444421/227236921-f097a8ff-a8d4-46c7-b05f-267bd63929fd.png)
